documentation/design-docs/dotnet-tools.md

```diff
@@ -396,6 +396,85 @@ CONVERT
       Writing:       ./trace.speedscope.json
       Conversion complete
 
+
+### dotnet-stack
+
+SYNOPSIS
+
+    dotnet-stack [options] [command] [<args>]
+
+OPTIONS
+
+    --version
+        Display the version of the dotnet-trace utility.
+
+    -h, --help
+        Show command line help
+
+COMMANDS
+
+    report         Displays stack traces for the target process
+
+REPORT
+
+    dotnet-stack report -p|--process-id <pid>
+                        [-t|--report-type <report-type>]
+                        [-h|--help]
+
+    Prints the managed stack from every thread in the target process
+
+    -t, ---report-type <report-type>
+        The way to represent the stack data.  (default: standard)
```

Comment:
I added this only since it gives us an easy place to add additional views of stack data, e.g., an Async stack view.

Reply:
Until we support more than one choice I'd suggest not creating the option.